Makefile

```diff
@@ -11,34 +11,34 @@ initdevdb:
 	sudo -u postgres psql -c "create database arlo with owner arlo;"
 
 install:
-	python3 -m pipenv install
+	python3.7 -m pipenv install
 	yarn install
 	yarn --cwd arlo-client install
 	yarn --cwd arlo-client build
 
 install-development:
-	python3 -m pipenv install --dev
+	python3.7 -m pipenv install --dev
 	yarn install
 	yarn --cwd arlo-client install
 
 resettestdb:
 	FLASK_ENV=test make resetdb
 
 resetdb:
-	python3 -m pipenv run python resetdb.py
+	python3.7 -m pipenv run python resetdb.py
 
 dev-environment: deps initdevdb install-development resetdb
 
 typecheck:
-	python3 -m pipenv run mypy .
+	python3.7 -m pipenv run mypy .
 
 format-python:
-	python3 -m pipenv run yapf --in-place --recursive .
+	python3.7 -m pipenv run black .
```

Comment:
Unrelated change to `black`.

Reply:
Ah, I see you addressed that in the description. The way I usually do that is to stack PRs, so you’d make this a PR based on that branch that targets that branch. Then you merge the first one, re-target this one, and merge it.

Reply:
Ah ok, that makes sense. I'll do that next time